src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-snapshot-form/rbd-snapshot-form.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="modal-header">
   <h4 class="modal-title float-left"
-      i18n>{ editing, select, true {Rename} other {Create}} RBD Snapshot</h4>
+      i18n>{ editing.toString(), select, true {Rename} other {Create}} RBD Snapshot</h4>
```

Comment:
In order for this to be aligned with the rest of forms, we should replace this with:
```suggestion
      i18n>{{ action | titlecase }} {{ resource | upperFirst }}</h4>
```
and populate `action` and 'resource` variables.

And same thing in the submit button:
```diff
- i18n>{ editing, select, true {Rename} other {Create}} Snapshot</cd-submit-button>
+ i18n>{{ action | titlecase }} {{ resource | upperFirst }}</cd-submit-button>
```

Reply:
I agree. Done.